packages/rich-text-links/src/index.ts

```diff
@@ -27,15 +27,20 @@ class Cache {
 }
 
 /**
- * Given a rich text document and an entity type, returns all unique matching
- * entity links.
+ * Given a rich text document and an (optional) entity type, returns all unique
+ * matching entity links.
+ *
+ * If there is no linkType provided, returns all unique entity links.
  */
-export function getRichTextEntityLinks(document: Document, linkType: EntityType): EntityLink[] {
+export function getRichTextEntityLinks(
+  document: Document,
+  linkType: EntityType | void = undefined,
```

Comment:
why not to use optional property instead:
```suggestion
  linkType?: EntityType,
```

Reply:
:+1: